simapp/utils.go

```diff
@@ -104,10 +105,10 @@ func CheckExportSimulation(
 }
 
 // PrintStats prints the corresponding statistics from the app DB.
-func PrintStats(db dbm.DB) {
-	fmt.Println("\nLevelDB Stats")
-	fmt.Println(db.Stats()["leveldb.stats"])
-	fmt.Println("LevelDB cached block size", db.Stats()["leveldb.cachedblock"])
+// TODO: implement stats collection for DBConnection
+func PrintStats(db dbm.DBConnection) {
```

Comment:
we should add a task for this, and make it require for v2release. @i-norden

Reply:
Created an issue: https://github.com/cosmos/cosmos-sdk/issues/11251